src/App.js

```diff
@@ -57,6 +57,18 @@ import TermsAndConditions from './screens/TermsAndConditions';
 import TxDetails from './screens/TxDetails';
 
 export default class App extends Component {
+
+  constructor(){
+    super();
+    if (__DEV__) {
+      YellowBox.ignoreWarnings([
```

Comment:
Not sure it's a good idea to ignore them. I think it's ok to keep them

Reply:
It is just annoying, have to click the `dismiss` button everytime, plus now we have cleaned our code and know that warnings only comes from the dependencies. Let's keep eyes on new code of `componentWill*` method, I think there is little probabilities that we will use these lifecycles methods since we now have React Hooks to use.